keyboards/czmao/a17_ds17r3/readme.md

```diff
@@ -0,0 +1,30 @@
+# mao ds 17r3 Righthand
+
+A 17keys numberpad keyboard with RGB backlight.
+This keyboard use 16mhz HSE and APM32F103CBT6(STM32F103CBT6) as MCU.
```

Comment:
> use 16mhz HSE

This is not currently configured for a 16MHz HSE, either

1. This statement is incorrect
2. The keyboard needs additional config